pallet-tfgrid/src/lib.rs

```diff
@@ -535,6 +536,28 @@ decl_module! {
             Ok(())
         }
 
+        #[weight = 10 + T::DbWeight::get().writes(1)]
+        pub fn delete_node_farm(origin, farm_id: u32, node_id: u32) -> dispatch::DispatchResult {
```

Comment:
Why is the farm_id passed? The node is loaded from db anyway, so if that is done first, the farm_id can be pulled from there and the input param is obsolote.

Reply:
mondays..